src/Analyzers/CSharp/Analyzers/RemoveUnreachableCode/CSharpRemoveUnreachableCodeDiagnosticAnalyzer.cs

```diff
@@ -38,12 +38,7 @@ protected override void InitializeWorker(AnalysisContext context)
 
         private void AnalyzeSemanticModel(SemanticModelAnalysisContext context)
         {
-            if (!(context.Options is WorkspaceAnalyzerOptions options))
-            {
-                return;
-            }
-
-            var fadeCode = options.Services.Workspace.Options.GetOption(FadingOptions.FadeOutUnreachableCode, LanguageNames.CSharp);
+            var fadeCode = context.GetOption(FadingOptions.FadeOutUnreachableCode, LanguageNames.CSharp);
```

Comment:
NOTE: FadingOptions are not editorconfig based. @sharwell do we already thread in the workspace options as a fallback to editorconfig options in the AnalyzerOptions passed to the analyzer when it executes in IDE? Otherwise, I can try to tackle that item next.

Reply:
> do we already thread in the workspace options as a fallback to editorconfig options in the AnalyzerOptions passed to the analyzer when it executes in IDE

No. NuGet-installed analyzers do not change behavior based on the host execution environment, so they must not have access to workspace options.